pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -142,6 +145,7 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		if !deployutil.IsTerminatedDeployment(latestDeployment) {
 			return c.updateStatus(config, existingDeployments)
 		}
+
 		return c.reconcileDeployments(existingDeployments, config)
 	}
 	// If the config is paused we shouldn't create new deployments for it.
```

Comment:
The block under the TODO has a return to explicitly skip new deployment... Am I missing something?